lib/index.js

```diff
@@ -12,7 +12,6 @@ var integration = require('@segment/analytics.js-integration');
 var json = require('json3');
 var keys = require('@ndhoule/keys');
 var localstorage = require('yields-store');
-var md5 = require('spark-md5').hash;
 var protocol = require('@segment/protocol');
```

Comment:
do we still need the uuid dependency?

Reply:
yep see https://github.com/segment-integrations/analytics.js-integration-segmentio/pull/55/files#diff-6d186b954a58d5bb740f73d84fe39073R477

Reply:
ah ok 👍